browser-test/src/question_lifecycle.test.ts

```diff
@@ -25,6 +26,7 @@ describe('normal question lifecycle', () => {
   })
 
   it('canonical question seeding works', async () => {
+    await dropTables(page)
```

Comment:
Can you add comment saying that this is needed when running on staging where it's not guaranteed that DB is empty when test starts?

Reply:
this was failing on local for me as well. it might be dependent on the order in which tests are run, so it's not always fails, but it's definitely not just staging issue.